hip_data_tools/apache/cassandra.py

```diff
@@ -32,6 +38,97 @@ def get_ssl_context(cert_path: str) -> SSLContext:
     return ssl_context
 
 
+_PYTHON_TO_CASSANDRA_DATA_TYPE_MAP = {
+    "Timestamp": "timestamp",
+    "str": "varchar",
+    "int64": "bigint",
+    "int32": "int",
+    "dict": "map",
+    "float64": "double",
+    "UUID": "UUID",
+}
+
+
+def _get_data_frame_column_types(data_frame):
+    data_frame_col_dict = {}
```

Comment:
why not use df.dtypes?
https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.dtypes.html

Reply:
that obfuscates most python datatypes as `object` this way allows you to peek into the values and find the correct datatype beyond object.